lists/malware.h

```diff
@@ -27,6 +27,22 @@ const std::vector<FilterList> malware_lists = {
     "https://disconnect.me/",
     "",
     ""
+  }), FilterList({
+    "E5484F0C-F35C-11E8-8EB2-F2801F1B9FD1",
+    "https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt",
+    "Malware Domain List",
+    {},
+    "http://www.malwaredomainlist.com/",
+    "",
+    ""
+  }), FilterList({
+    "237B68E0-F35D-11E8-8EB2-F2801F1B9FD1",
+    "https://mirror.cedia.org.ec/malwaredomains/justdomains",
+    "Malware domains",
+    {},
+    "https://www.malwaredomains.com/",
```

Comment:
Many of the [mirrors they list on their website](https://www.malwaredomains.com/?page_id=29) are out-of-date but the [one used in this pull request](https://mirror.cedia.org.ec/malwaredomains/) seems seems to be fresh (last update 2019-02-20).

They used to post about list updates and false positives on their [homepage](https://www.malwaredomains.com/) but the [last update post](https://www.malwaredomains.com/?p=4166) was on 2018-03-08 and the [last false positive announcement](https://www.malwaredomains.com/?p=4172) was on 2018-06-08.

That said, they appear to still be [adding](https://mirror.cedia.org.ec/malwaredomains/20190220.txt) and [removing](https://mirror.cedia.org.ec/malwaredomains/removed-domains-20190220.txt) domains from their list.